content/en/docs/reference/kubectl/kubectl.md

```diff
@@ -69,14 +62,7 @@ kubectl [flags]
     </tr>
 
     <tr>
-      <td colspan="2">--boot-id-file string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/proc/sys/kernel/random/boot_id"</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of files to check for boot-id. Use the first one that exists.</td>
-    </tr>
-
-    <tr>
-      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "~/.kube/http-cache"</td>
+      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/Users/dsatya/.kube/http-cache"</td>
```

Comment:
better to have something like `/home/username/.kube...`

Reply:
Any idea how this username was inserted into the output? Perhaps check the proposed output against the current (v1.16) page.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go#L57
and
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go#L326

Reply:
@neolit123 Can I apply this manually?

Reply:
@daminisatya yes